oqum/risk/02_config_event_based_risk.tex

```diff
@@ -0,0 +1,53 @@
+\item  \Verb+risk_investigation_time+: this parameter specifies the time period for which the event loss tables and loss exceedance curves will be calculated. If this parameter is not provided in the risk job configuration file, the time period used is the same as that specifed in the hazard calculation;
+\item  \Verb+conditional_loss_poes+: this parameter is used to define the probabilities of exceedance at which loss maps are to be produced;
+\item  \Verb+sites_disagg+: list of locations (pairs of longitude and latitude) where the loss disaggregation should be carried out. Notice that in order to perform the loss disaggregation, assets needs to exist at those locations;
+\item  \Verb+mag_bin_width+: this parameter specifies the with of the magnitude bins (in Mw);
+\item  \Verb+distance_bin_width+: this parameter specifies the with of the distance bins (in km);
+\item  \Verb+coordinate_bin_width+: this parameter specifies the with of the coordinates bins (in decimal degrees);
+\item  \Verb+specific_assets+: this parameter specifies the set of assets for which the asset event loss table will be generated;
+\end{itemize}
+
+The definition of the parameters for the loss disaggregation follow the same rules established for the seismic hazard disaggregation described in section~\ref{subsec:config_hazard_disaggregation}.
+
+\paragraph{Note regarding the new event-based risk calculators}
+
+Starting from this release some of the scientific calculators are saving their inputs and outputs in a single HDF5 file, called the datastore. The HDF5 format is a well known standard in the scientific community, can be read/written by a variety of programming languages and with different tools and it is a state-of-the-art technology when it comes to managing large numeric datasets. The change to the HDF5 technology provides huge performance benefits compared to the earlier approach used by the engine, which involved storing arrays in PostgreSQL.
+
+In OpenQuake 1.5 the event based calculators based on Postgres (both hazard and risk) are officially deprecated. They are still present and work as before, but they are being replaced with new versions of the calculators based on the HDF5 technology. The actual removal of the old calculators is scheduled for OpenQuake 1.6. The change will have no impact on regular users, who will simply notice a definite improvement in erformance. Nonetheless, the change will affect power users who are performing queries on the OpenQuake database, since there will be nothing left in the database once we remove the old calculators.
```

Comment:
This section was correct in OpenQuake 1.5 but now the old calculators have been removed. I would just remove the en tire section *Note regarding the new event-based risk calculators*